pkgcfg/pkgcfg.go

```diff
@@ -51,16 +26,22 @@ func CreateFiles(pkg, basePath string) error {
 	}
 	for name, contentPath := range cfg.CreateFiles {
 		filePath := filepath.Join(basePath, name)
-		if _, err := os.Stat(filePath); err == nil {
-			continue
-		}
-		content, err := os.ReadFile(filepath.Join(basePath, contentPath))
+		// if _, err := os.Stat(filePath); err == nil {
+		// 	continue
+		// }
+		content, err := os.ReadFile(filepath.Join(cfg.localConfigPath, contentPath))
 		if err != nil {
 			return errors.WithStack(err)
 		}
+		if err = createDir(filepath.Dir(filePath)); err != nil {
+			return err
+		}
 		if err := os.WriteFile(filePath, content, 0744); err != nil {
```

Comment:
Not for this PR but something to think about - it sounds like we'll eventually need a way to specify the file's permissions. I'm guessing not all of them should be executable.